### ISSUE TITLE
TTL not work until not upload files

### ISSUE THREAD

**Issue by xiaoweihong** [chrislusf/seaweedfs#992]
when i use seaweedfs, i use the ttl function.But i find the ttl not work until i don't upload files to weedfs

**Comment by chrislusf** [comment]
Please follow the issue template.

**Comment by xiaoweihong** [comment]
Sponsors SeaweedFS via Patreon https://www.patreon.com/seaweedfs

**Describe the bug**
TTL not work until not upload files

**System Setup**
List the command line to start "weed master", "weed volume", "weed filer", "weed s3", "weed mount".
server
```
./weed -logtostderr  master -ip=$IP -port=$masterPort -mdir=$masterDir -metrics.address=127.0.0.1:9091 --volumeSizeLimitMB 30000
```
volume
```
./weed -logtostderr volume -mserver=$IP:9333  -ip=$IP -dir=$dataDir -max=60 -port=9301 -read.redirect=true
```
upload file
```
curl http://192.168.100.166:9333/dir/assign?ttl=1m
curl -F file=@test.txt 192.168.100.166:9301/63,04e861d9bb90
```
when the upload is consistent like above.When 1 minute goes by,the file don't be deleted

**Comment by chrislusf** [comment]
the file ttl and volume ttl can be different. Currently the file upload also need ttl.

Added a fix c2960c7 to default to volume ttl
